src/helloworld.js

```diff
@@ -0,0 +1,10 @@
+import React, { Component } from 'react';
+
+class Hello extends React.Component {
+  render() {
+    return <h1>Hello {this.props.name}!</h1>;
```

Comment:
You are rendering `this.props.name` here, but in App.js, the prop `name` is never passed.